tests/pyfunc/test_pyfunc_model_with_type_hints.py

```diff
@@ -731,3 +731,58 @@ def predict(self, model_input: list[str]) -> list[str]:
         # has the package, this could cause version not found error
         env_manager=VIRTUALENV,
     )
+
+
+def test_predict_with_wrong_signature_warns():
+    message = r"The underlying model's `predict` method contains invalid parameters"
```

Comment:
`The udnerlying model's` doesn't sound right. Do we need it?

Reply:
Ah right, I moved this message from the wrapper :( Let me update